### ISSUE TITLE
test set of paradetox

### ISSUE THREAD

**Issue by GenderBiasVL** [shengliu66/ICV#10]
I would like to know whether the test set used is the Paradetox 671 test set, which requires obtaining permission from Paradetox authors, or if it was sampled from the training set.

**Comment by sumingfirst** [comment]
你的邮件我受到非常感谢配合
